styles.css

```diff
@@ -136,9 +136,10 @@ img {
 .box-footer {
   background-color: #5356A4;
   height: 40px;
+  display: flex;
+  align-items: center;
+  padding: 0px 20px;
 }
 .box-footer h3 {
   color: #FFFFFF;
-  padding: 0px 20px;
-  display: flex;
```

Comment:
Do we want to use pixels for this padding because it is so small? Won't affect the Desktop layout I assume?